### ISSUE TITLE
[CoE Starter Kit - BUG] Customizing Child Flow

### ISSUE THREAD

**Issue by tomsinsight** [microsoft/coe-starter-kit#3078]
### Describe the issue

Hello Team,

I have successfully deployed latest CoE version and require to extend the some of the Gov flows. when I try to save one of the child flow, it throws following error:
![image](https://user-images.githubusercontent.com/98403757/176221601-c1960474-59d2-4005-8709-466496b73aa3.png)

I assume this could be the known bug at Power Platform end, however I was wondering - in this case, should I create the flow from the scratch to customize it? or there is any other alternative that you may suggest?

Thank you for your time in advance!

### Expected Behavior

Child flow should be saved succefully.

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

2.17

### What app or flow are you having the issue with?

HELPER - ObjectOperations Gov

**Comment by Jenefer-Monroe** [comment]
Yes unfortunately this is a product limitation: https://github.com/microsoft/coe-starter-kit/issues/2284
If you post the suggestions they may be things we can add to the kit though if they are helpful to a broad set of admins.
